guide/14-deep-learning/object-detection.ipynb

```diff
@@ -69,7 +69,7 @@
        "                        </a>\n",
```

Comment:
<ol><li>paragraph could be started by this line --</li></ol><blockquote>Now, we can further deploy the trained model to run prediction on a video. <code>predict_video</code>&nbsp;function is used to run prediction on a video and</blockquote><p> </p><p> </p><p><em>You can select assignment threshold for the assignment of trackers, the number of frames the detection should persist if object is not found and threshold of number of frames to start detecting an object. These values can be passed as a dictionary in the argument tracking options.</em></p><p> </p><p>this above could be elaborated a bit and rephrased as : -</p><p> </p><blockquote>The following options/parameters can be selected in the predict video function by the user : -</blockquote><p> </p><blockquote>Assignment threshold <strong>- </strong> There might be multiple trackers detecting and tracking objects. The Intersection over Union (iou) threshold can be set to assign a tracker with the mentioned threshold value.</blockquote><blockquote>Then the number of frames the object remains absent from the frame to be considered as vanished.</blockquote><blockquote>Also the number of frames an object remains present in the frame to start tracking it.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>